test/e2e/cmd/run/run.go

```diff
@@ -577,36 +569,6 @@ func (h *helper) createKubeClient() (*kubernetes.Clientset, error) {
 	return kubernetes.NewForConfig(config)
 }
 
-// monitorTestJob keeps track of the test pod to determine whether the tests failed or not.
-func (h *helper) startAndMonitorTestJobs(client *kubernetes.Clientset) error {
-	testSession := NewJobsManager(client, h)
-
-	outputs := []io.Writer{os.Stdout}
-	if h.logToFile {
-		jl, err := newJSONLogToFile(fmt.Sprintf(testsLogFilePattern, h.testContext.ClusterName))
-		if err != nil {
-			log.Error(err, "Failed to create log file for test output")
-			return err
-		}
-		defer jl.Close()
-		outputs = append(outputs, jl)
-	}
-	writer := io.MultiWriter(outputs...)
-	runJob := NewJob("eck-"+h.testRunName, "config/e2e/e2e_job.yaml", writer, goLangTestTimestampParser)
-
-	if h.deployChaosJob {
```

Comment:
This removes the ability to deploy the chaos job for the resilience nightly test.